app/templates/issues.hbs

```diff
@@ -23,11 +23,7 @@
       {{#card.title class="flex-nogrow"}}
         <h3>{{issue.title}}</h3>
       {{/card.title}}
-      {{#card.content class="layout-column flex-grow"}}
-        {{#read-more class="flex-nogrow" maxHeight="350px"}}
-          {{markdown-to-html markdown=issue.body extensions='issueWrap'}}
```

Comment:
Why is this misteriously removed?

Reply:
@blazeu The readmore package is not updated for a while, caused deprecation warning.

I removed the entire file in https://github.com/bekicot/git-task-list/tree/use-graphql anyway.